### PR TITLE
swap  broken link

### PR DESCRIPTION
**Pull request by thefrankharvey**
## Description
- The link on this page for 'queried with GraphQL' was returning a Page Not Found.
- From the DOM "\<a href="/docs/querying-with-graphql/" class="css-0">queried with GraphQL</a>"
- In the mdx file I swapped the link from `/docs/querying-with-graphql/` to this working link `/docs/page-query/`

### CODE REVIEW

File: docs/docs/mdx/writing-pages.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ date: 2019-01-29
 # Hello, world!
 ```
 
-Which can then be [queried with GraphQL](/docs/querying-with-graphql/):
+Which can then be [queried with GraphQL](/docs/page-query/):
```
Comment (pieh) [inline docs/docs/mdx/writing-pages.md]: Given this redirect: https://github.com/gatsbyjs/gatsby/blob/30437865257c21498b6da0df026e67d2983bfa2c/www/gatsby-node.js#L368-L372

Should we change link to `/docs/graphql-concepts/`?

Comment (thefrankharvey) [reply docs/docs/mdx/writing-pages.md]: Good call. Didn't think to check the gatsby-node file. Will update to `/docs/graphql-concepts/`

Comment (pieh) [reply docs/docs/mdx/writing-pages.md]: I navigated to old url directly and saw that I was redirected and then found this code (I wouldn't start by looking there :P)

Comment (thefrankharvey) [reply docs/docs/mdx/writing-pages.md]: Oh good, thanks for the reassurance! I changed it to '/docs/graphql-concepts/' and commited again.

**Pull request merged**